lib/network/tsnetwork_select.c

```diff
@@ -46,9 +46,6 @@ network_select(int blocking)
 		return (events_run());
 	}
 
```

Comment:
This should have `/* UNREACHABLE */` added, just for the sake of clarity.  Sorry I didn't notice this on the first pass.